packages/reporter/cypress/support/index.ts

```diff
@@ -3,6 +3,9 @@ import 'cypress-real-events/support'
 import installCustomPercyCommand from '@packages/ui-components/cypress/support/customPercyCommand'
 
 installCustomPercyCommand({
+  before () {
+    cy.get('.focus-tests-text').should('be.visible')
```

Comment:
The header should always display [< Tests] and not just [<] and all snapshots include the header, so this makes sure it's visible before every snapshot.